### ISSUE TITLE
BUG report: update text of shape changes the font size

### ISSUE THREAD

**Issue by sergedc** [ShapeCrawler/ShapeCrawler#638]
In my PPT I have a shape with text in font size 15. 
After I do:

Debug.WriteLine(shape.TextFrame.Paragraphs.First().Portions.First().Font.Size);
shape.TextFrame.Text = "A new shape text";
Debug.WriteLine(shape.TextFrame.Paragraphs.First().Portions.First().Font.Size);

The shape's text then becomes font size 11.

Console output: 
15
11

**Comment by jcoliz** [comment]
Yeah, this is troublesome. It also changes the font family name as well for me. The workaround I have found is to not change the textframe text, but rather change only the paragraph text. Then I just make sure there are as many paragraphs already in the source presentation that I'll need.

**Comment by jcoliz** [comment]
There is some suspicious stuff going on in `FontService.GetAdjustedFontSize`.

At the top of the method:

```c#
        paint.TextSize = fontSize;
```

And then at the bottom:

```c#
        const int dpi = 96;
        var points = Math.Round(paint.TextSize * 72 / dpi, 0);
        return (int)points;
```

Ergo, text size is guaranteed to get a 25% cut, even if the `wordWidth` never hits the `rect.Right`.

**Comment by jcoliz** [comment]
I now think the problem is that `FontService.GetAdjustedFontSize` sets the `SKPaint.TextSize` (in pixels) directly to the font size )in points), without making a points-to-pixels conversion. Once the previous PR gets through, I'll submit this for review.

**Comment by ashahabov** [comment]
@sergedc, it works correctly for me. Could you please check the latest library version.
